src/emulator/main.cpp

```diff
@@ -154,9 +159,19 @@ int main(int argc, char *argv[]) {
         }
         gui::draw_end(host.window.get());
 
-        if (!host.cfg.sync_rendering)
+        if (!host.cfg.sync_rendering) {
             host.display.condvar.notify_all();
-        else {
+            if (host.cfg.fps_limit) {
+                const std::chrono::microseconds TARGET_TIME{ static_cast<uint32_t>(1000000.0 / (host.cfg.desired_fps+1)) };
```

Comment:
```suggestion
                const std::chrono::microseconds TARGET_TIME{ static_cast<uint32_t>(1000000.0 / (host.cfg.desired_fps + 1)) };
```

This is not really necessary but it looks more nice. Also it's more great if you can name the 1000000.0 (to an constexpr maybe? for future in case anyone want to understand the code)